src/services/AudiusBackend.js

```diff
@@ -1041,25 +1055,6 @@ class AudiusBackend {
     }
   }
 
-  static async getTrendingTracks({ genre, offset, limit, timeRange }) {
```

Comment:
@raymondjacobson we are getting rid of the optimistic load with this change, just wanted to flag. Should be fine if dp selection isn't slow.

Reply:
nvm there is eager selection in the new api 😍  thx @piazzatron